test/integration/user_can_remove_items_from_cart_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_helper'
+
+class UserCanEditItemQuantityInCart < ActionDispatch::IntegrationTest
+
+  test "user removes item from cart by subtracting until quantity is 0" do
+    item = create(:item)
+
+    visit items_path
+    click_link "Add to Duffel"
+    visit duffel_path
+
+    within '.cart-item-info' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.